src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -453,7 +455,7 @@ public virtual JwtSecurityToken CreateJwtSecurityToken(string issuer, string aud
         /// <exception cref="ArgumentException">If 'expires' &lt;= 'notBefore'.</exception>
         public virtual JwtSecurityToken CreateJwtSecurityToken(string issuer = null, string audience = null, ClaimsIdentity subject = null, DateTime? notBefore = null, DateTime? expires = null, DateTime? issuedAt = null, SigningCredentials signingCredentials = null)
```

Comment:
Do we need to add a input parameter in CreateJwtSEcurityToken to provide an option to pass in SecurityTokenDescriptor.Claims?